Lang/French.pj.Lang

```diff
@@ -316,59 +316,59 @@
 
 //Save Type
 #620#  "Par défaut"
-#621#  "4-kbit EEPROM"
-#622#  "16-kbit EEPROM"
-#623#  "32-kbyte SRAM"
+#621#  "4 kbit EEPROM"
+#622#  "16 kbit EEPROM"
+#623#  "32 ko SRAM"

```

Comment:
This is the only change I didn't know about, because in English I know the hyphen (`'-'`) is required.

I don't know if that's ever appropriate for French, though, but Google Translate suggested "32-kbyte SRAM" to "32 Ko de SRAM", so I guess you're probably right.